packages/e2e-test-utils/src/set-option.js

```diff
@@ -11,19 +11,23 @@ import { pressKeyWithModifier } from './press-key-with-modifier';
  *
  * @param {string} setting The option, used to get the option by id.
  * @param {string} value   The value to set the option to.
+ *
  */
 export async function setOption( setting, value ) {
 	await switchUserToAdmin();
-	await visitAdminPage( 'options-general.php' );
+	await visitAdminPage( 'options.php' );
+
+	if ( typeof value === 'boolean' ) {
+		value = value ? '1' : '0';
+	}
```

Comment:
Do we need this check?

On the field, is still 1 or 0. Both for getting and receiving that value. If we put `setOption('option', '1')` instead of `setOption('option', true)` we would not need to make the conversion here and also on the test file.

Is the checkbox working with the values for real or not? I'm not yet pretty sure about that.

Reply:
Yeah, it's really just syntactic sugar at this point, and there's a chance that it's more confusing than helpful to people.

We can remove this if you want -- it just means that people must remember to pass a `'1'` or `'0'` (string!) 😅

Reply:
Especially since we're not doing the same in `getOption` 🤔  Okay, you have convinced me, I'll remove it.

Reply:
It is tricky, to be honest. But WordPress database is saving 1 or 0 rather than true or false I guess.

Reply:
Yeah, and it makes everything much more consistent -- e.g. restoring a previous value that was obtained from `getOption`.